draft-irtf-cfrg-aead-limits.md

```diff
@@ -183,22 +191,24 @@ This document defines limitations in part using the quantities below.
 | k | AEAD key length (in bits) |
 | r | AEAD nonce length (in bits) |
 | t | Size of the authentication tag (in bits) |
-| l | Length of each message (in blocks) |
+| l | Maximum length of each message (in blocks) |
 | s | Total plaintext length in all messages (in blocks) |
 | q | Number of protected messages (AEAD encryption invocations) |
 | v | Number of attacker forgery attempts (failed AEAD decryption invocations) |
-| p | Adversary attack probability |
-| o | Offline adversary work (in number of encryption and decryption queries; multi-user setting only) |
-| u | Number of users or keys (multi-user setting only) |
-| B | Maximum number of blocks encrypted by any user or key (multi-user setting only) |
+| p | Upper bound on adversary attack probability |
+| o | Offline adversary work (in number of encryption and decryption queries; multi-key setting only) |
+| u | Number of keys (multi-key setting only) |
+| B | Maximum number of blocks encrypted by any key (multi-key setting only) |
 
 For each AEAD algorithm, we define the (passive) confidentiality and (active)
 integrity advantage roughly as the advantage an attacker has in breaking the
-corresponding classical security property for the algorithm. Moreover, we 
-define the combined authenticated encryption advantage guaranteeing both
-confidentiality and integrity against an active attacker. Specifically:
+corresponding classical security property for the algorithm. A passive attacker
+can query ciphertexts for arbitrary plaintexts. An active attacker can additionally
+query plaintexts for arbitrary ciphertexts. Moreover, we define the combined
+authenticated encryption advantage guaranteeing both confidentiality and integrity
+against an active attacker. Specifically:
 
-- Confidentiality advantage (CA): The probability of a passive attacker
+- Confidentiality advantage (CA): The probability of a passive attacker capable of
 succeeding in breaking the confidentiality properties (IND-CPA) of the AEAD scheme.
```

Comment:
"the probability of an attempted attack succeeding" might be better

Reply:
I kept this as-is to be consistent with the IA definition. (We can rework in a future issue/PR if needed.)